vision_datasets/visual_question_answering/vqa_as_kvp_dataset.py

```diff
@@ -0,0 +1,99 @@
+import logging
+from copy import deepcopy
+from typing import Any, Dict
+
+from vision_datasets.common import DatasetTypes, KeyValuePairDatasetInfo, VisionDataset
+from vision_datasets.key_value_pair import (
+    KeyValuePairDatasetManifest,
+    KeyValuePairLabelManifest,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class VQAAsKeyValuePairDataset(VisionDataset):
+    """Dataset class that access Visual Question Answering (VQA) datset as KeyValuePair dataset."""
+
+    ANSWER_KEY = "answer"
+    RATIONALE_KEY = "rationale"
+    QUESTION_KEY = "question"
+    SCHEMA_BASE = {
+        "name": "Visual Question Answering",
+        "description": "Answer questions on given images and provide rationale for the answer.",
```

Comment:
update description accordingly if rationale is not available / needed

Reply:
updated